test/test_api/test_api.py

```diff
@@ -449,3 +451,92 @@ def test_do_dummy_prediction(dask_client, fit_dictionary_tabular):
     estimator._clean_logger()
 
     del estimator
+
+
+@pytest.mark.parametrize("disable_file_output", [True, False])
+@pytest.mark.parametrize('openml_id', (40981,))
```

Comment:
Can we test another configuration :), I think we always test Australian... Maybe add:

- https://www.openml.org/d/21
- https://www.openml.org/d/40984
- https://www.openml.org/d/1066

Why are very fast?

Reply:
What do you mean by why are very fast? Yes, I'll put one of these configuration